Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/ScreenCapturer.swift

```diff
@@ -34,9 +34,16 @@ internal class ScreenCapturer {
             return
         }
 
+        Task {
+            await doCapture(window: window, authorization: authorization, captureUI: captureUI)
+        }
+    }
+
+    @MainActor
+    private func doCapture(window: UIWindow?, authorization: Authorization, captureUI: ScreenCaptureUI) async {
```

Comment:
I split this to a helper function only to avoid a noisier diff caused by indenting everything a level.